server.js

```diff
@@ -1,10 +1,12 @@
 let http = require('http');
 const routes = require(`./router`);
+
 require('dotenv').config();
 // ******** Http Server **************//
 http.createServer(routes).listen(process.env.PORT);
 console.log("server running on port 3000");
```

Comment:
`server running on port 3000`? the port number should be dynamic as it's an env variable.
I'd have done something like this-

```
const port = process.env.PORT || 3000;
...
console.log(`server is listening on ${port}`);
...
```